src/java.base/share/classes/sun/security/pkcs/PKCS8Key.java

```diff
@@ -218,20 +218,13 @@ public String getFormat() {
     private synchronized byte[] getEncodedInternal() {
         if (encodedKey == null) {
             try {
-                DerOutputStream out = new DerOutputStream ();
                 DerOutputStream tmp = new DerOutputStream();
```

Comment:
What is the criteria of using the default constructor vs the one with a initial size? Here is using the default, are we sure about the key will always fit?

Reply:
Here the key is the last thing to be written into the DerOutputStream, so there will be no more reallocation after and its content will not be leaked.

Reply:
I see. Interesting...